draft-ietf-tls-tls13.md

```diff
@@ -2369,7 +2369,7 @@ If the same extension appears in both locations, the client
 MUST rely only on the value in the EncryptedExtensions block.
 [[OPEN ISSUE: Should we just produce a canonical list of what
 goes where and have it be an error to have it in the wrong
-place? That seems simpler. Perhaps have a whitelist of which
+place? That seems simpler. Perhaps have a white list of which
```

Comment:
This is correct as-is. "whitelist" is a compound word.
https://en.wikipedia.org/wiki/Whitelist